correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -37,40 +77,47 @@ Spaces are allowed between properties and before and after equal sign. Propertie
 
 # Examples of HTTP headers
 
-Single header: 
+Single header:
 
 ```
-Correlation-Context: userId=sergey,serverNode=DF:28,isProduction=false
+Correlation-Context: v=0,userId=sergey,serverNode=DF:28,isProduction=false
 ```
 
 Context might be split into multiple headers:
 
 ```
-Correlation-Context: userId=sergey
-Correlation-Context: serverNode=DF%3A28,isProduction=false
+Correlation-Context: v=0,userId=sergey
+Correlation-Context: v=1,serverNode=DF%3A28,isProduction=false
+```
+
+Context might be re-combined into a single header:
+
+```
```

Comment:
this is a good example for the versioning section. Let's not use `v=1` in the examples section - it gives the wrong impression